dts/bindings/display/ilitek,ili9342c.yaml

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) 2018, Jan Van Winkel <jan.van_winkel@dxplore.eu>
```

Comment:
I've commented about not using the default values on a few registers in this file. In order to avoid making too many comments, the same thing applies to any register in this file. It should follow the default in the datasheet unless there is a good reason to deviate.